src/overlays/actors/ovl_En_Bomjima/z_en_bomjima.c

```diff
@@ -45,83 +83,1070 @@ const ActorInit En_Bomjima_InitVars = {
     (ActorFunc)EnBomjima_Draw,
 };
 
-// static ColliderCylinderInit sCylinderInit = {
-static ColliderCylinderInit D_80C00A18 = {
-    { COLTYPE_NONE, AT_NONE, AC_NONE, OC1_ON | OC1_TYPE_PLAYER, OC2_TYPE_2, COLSHAPE_CYLINDER, },
-    { ELEMTYPE_UNK0, { 0x00000000, 0x00, 0x00 }, { 0xF7CFFFFF, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_NONE, OCELEM_ON, },
+static ColliderCylinderInit sCylinderInit = {
+    {
+        COLTYPE_NONE,
+        AT_NONE,
+        AC_NONE,
+        OC1_ON | OC1_TYPE_PLAYER,
+        OC2_TYPE_2,
+        COLSHAPE_CYLINDER,
+    },
+    {
+        ELEMTYPE_UNK0,
+        { 0x00000000, 0x00, 0x00 },
+        { 0xF7CFFFFF, 0x00, 0x00 },
+        TOUCH_NONE | TOUCH_SFX_NORMAL,
+        BUMP_NONE,
+        OCELEM_ON,
+    },
     { 10, 30, 0, { 0, 0, 0 } },
 };
 
-#endif
+static u16 D_80C00A44[] = {
+    0x0719,
+    0x071A,
+    0x071B,
+    0x0708,
+};
+
+static u16 D_80C00A4C[] = {
+    0x0739,
+    0x073A,
+    0x073B,
+    0x0000,
+};
+
+static u16 D_80C00A54[] = { 0x0739, 0x073A, 0x073B, 0x0714, 0x0709, 0x070A, 0x070B,
+                            0x070C, 0x070D, 0x070E, 0x070F, 0x0712, 0x0713, 0x0000 };
+
+static u16 D_80C00A70[] = { 0x0739, 0x073A, 0x073B, 0x0759, 0x0753, 0x0754, 0x0755,
+                            0x0756, 0x070D, 0x0757, 0x0758, 0x0712, 0x0713, 0x0000 };
```

Comment:
```suggestion
static u16 D_80C00A54[] = { 0x0739, 0x073A, 0x073B, 0x0714, 0x0709, 0x070A, 0x070B,
                            0x070C, 0x070D, 0x070E, 0x070F, 0x0712, 0x0713 };

static u16 D_80C00A70[] = { 0x0739, 0x073A, 0x073B, 0x0759, 0x0753, 0x0754, 0x0755,
                            0x0756, 0x070D, 0x0757, 0x0758, 0x0712, 0x0713 };
```
The max index this can have looks to be 12. So this is padding